### PR TITLE
Acronym Expansion For Clearer Translation

### PR DESCRIPTION
**Pull request by TokyoScarab**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/4270

### CODE REVIEW

**Comment by jvsam** [comment]
@nenonix and @JohanFreelancer9 for copy review please. Thanks.

File: windows/deployment/update/waas-delivery-optimization-reference.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ Download mode dictates which download sources clients are allowed to use when do
...
+| Group (2) | When group mode is set, the group is automatically selected based on the device’s Active Directory Domain Services (AD DS) site (Windows 10, version 1607) or the domain the device is authenticated to (Windows 10, version 1511). In group mode, peering occurs across internal subnets, between devices that belong to the same group, including devices in remote offices. You can use GroupID option to create your own custom group independently of domains and Active Directory Domain Services sites. Starting with Windows 10, version 1803, you can use the GroupIDSource parameter to take advantage of other method to create groups dynamically. Group download mode is the recommended option for most organizations looking to achieve the best bandwidth optimization with Delivery Optimization. |
```
Comment (JohanFreelancer9) [inline windows/deployment/update/waas-delivery-optimization-reference.md]:
```suggestion
| Group (2) | When group mode is set, the group is automatically selected based on the device’s Active Directory Domain Services (AD DS) site (Windows 10, version 1607) or the domain the device is authenticated to (Windows 10, version 1511). In group mode, peering occurs across internal subnets, between devices that belong to the same group, including devices in remote offices. You can use the GroupID option to create your own custom group independently of domains and Active Directory Domain Services sites. Starting with Windows 10, version 1803, you can use the GroupIDSource parameter to take advantage of other methods to create groups dynamically. Group download mode is the recommended option for most organizations looking to achieve the best bandwidth optimization with Delivery Optimization. |
```

Comment (jvsam) [reply windows/deployment/update/waas-delivery-optimization-reference.md]: Hi @TokyoScarab can you please check @JohanFreelancer9 's feedback? Thanks.

Comment (jvsam) [reply windows/deployment/update/waas-delivery-optimization-reference.md]: Hi again @TokyoScarab please check the changes so we can move to the next step to have this PR approved for merging. Thanks.

**Comment by jvsam** [comment]
Hi @greg-lindsay this pull request has been copy edited and is ready for your final review and approval. Please let us know if there are outstanding issues to be resolved prior to merge. Thank you very much.

File: windows/deployment/update/waas-delivery-optimization-reference.md [hunk 2]
```diff
@@ -116,7 +116,7 @@ Download mode dictates which download sources clients are allowed to use when do
...
+By default, peer sharing on clients using the group download mode is limited to the same domain in Windows 10, version 1511, and the same domain and Active Directory Domain Services site in Windows 10, version 1607. By using the Group ID setting, you can optionally create a custom group that contains devices that should participate in Delivery Optimization but do not fall within those domain or Active Directory Domain Services site boundaries, including devices in another domain. Using Group ID, you can further restrict the default group (for example, you could create a sub-group representing an office building), or extend the group beyond the domain, allowing devices in multiple domains in your organization to be peers. This setting requires the custom group to be specified as a GUID on each device that participates in the custom group.
```
Comment (illfated) [inline windows/deployment/update/waas-delivery-optimization-reference.md]: Minor typo from before this PR, plural S missing in "within those domain".
```suggestion
By default, peer sharing on clients using the group download mode is limited to the same domain in Windows 10, version 1511, and the same domain and Active Directory Domain Services site in Windows 10, version 1607. By using the Group ID setting, you can optionally create a custom group that contains devices that should participate in Delivery Optimization, but do not fall within those domains or Active Directory Domain Services site boundaries, including devices in another domain. Using Group ID, you can further restrict the default group (for example, you could create a sub-group representing an office building), or extend the group beyond the domain, allowing devices in multiple domains in your organization to be peers. This setting requires the custom group to be specified as a GUID on each device that participates in the custom group.
```
Also added a comma before "but do not fall within those domains", to separate the subordinate clause from its main sentence.

Comment (jvsam) [reply windows/deployment/update/waas-delivery-optimization-reference.md]: Thanks for catching that @illfated .

Can you please check @TokyoScarab ?

File: windows/deployment/update/waas-delivery-optimization-reference.md [hunk 3]
```diff
@@ -106,7 +106,7 @@ Download mode dictates which download sources clients are allowed to use when do
...
+| Group (2) | When group mode is set, the group is automatically selected based on the device’s Active Directory Domain Services (AD DS) site (Windows 10, version 1607) or the domain the device is authenticated to (Windows 10, version 1511). In group mode, peering occurs across internal subnets, between devices that belong to the same group, including devices in remote offices. You can use the GroupID option to create your own custom group independently of domains and Active Directory Domain Services sites. Starting with Windows 10, version 1803, you can use the GroupIDSource parameter to take advantage of other methods to create groups dynamically. Group download mode is the recommended option for most organizations looking to achieve the best bandwidth optimization with Delivery Optimization. |
```
Comment (nenonix) [inline windows/deployment/update/waas-delivery-optimization-reference.md]:
```suggestion
| Group (2) | When group mode is set, the group is automatically selected based on the device’s Active Directory Domain Services (AD DS) site (Windows 10, version 1607), or the domain the device is authenticated to (Windows 10, version 1511). In group mode, peering occurs across internal subnets, between devices that belong to the same group, including devices in remote offices. You can use the GroupID option to create your own custom group independently of domains and Active Directory Domain Services sites. Starting with Windows 10, version 1803, you can use the GroupIDSource parameter to take advantage of other methods to create groups dynamically. Group download mode is the recommended option for most organizations looking to leverage bandwidth with Delivery Optimization. |
```

Comment (jvsam) [reply windows/deployment/update/waas-delivery-optimization-reference.md]: ^^ Please check @TokyoScarab . Thanks.

**Comment by jvsam** [comment]
Thank you so much @Dansimp !

**Pull request merged**